apps/web/scripts/generate-cms-dictionaries.ts

```
@@ -0,0 +1,65 @@
+import {
+  getArticleSections,
+  getBlogCategoryLandingData,
+  getBlogArticleData,
+} from "@/services/datocms";
+
+import { AVAILABLE_LANGS } from "@/lib/config";
+import { Lang } from "@/dictionaries/locales";
+import { generateShortLink } from "@/app/actions/general";
+import path from "node:path";
+import * as fs from "fs";
```

Biome

[error] A Node.js builtin module should be imported with the node: protocol.

Using the node: protocol is more explicit and signals that the imported module belongs to Node.js.
Unsafe fix: Add the node: protocol.


(lint/style/useNodejsImportProtocol)